config/Config-images.in

```diff
@@ -224,6 +224,14 @@ menu "Target Images"
 		help
 		  If you don't know, 5 seconds is a reasonable default.
 
+	config EFI_IMAGES
+		bool "Build EFI GRUB images (Linux x86 or x86_64 host only)"
+		depends on TARGET_x86_64 || (TARGET_x86 && !TARGET_x86_rdc)
+		depends on TARGET_ROOTFS_EXT4FS || TARGET_ROOTFS_ISO || TARGET_ROOTFS_JFFS2 || TARGET_ROOTFS_SQUASHFS
+		select GRUB_IMAGES
```

Comment:
PR splitted in two commit : 
- For sgdisk tools : https://github.com/openwrt/openwrt/pull/391/commits/3871e7c5c1e5e3beff8127e45d7d2f70dff174d7
- For EFI Grub : https://github.com/openwrt/openwrt/pull/391/commits/a9b15e43862459f45c75edf10aac5f82e150043a